docs/tasks.md

```diff
@@ -439,4 +439,256 @@ The following tables gives you an overview of the tasks in MTEB.
+
+<br/>
+
+## Task Count per Language
+Note: Cross lingual tasks are omitted in this table.
+
+<!-- TASK LANG TABLE START -->
+| Language | BitextMining | Classification | Clustering | InstructionRetrieval | MultilabelClassification | PairClassification | Reranking | Retrieval | STS | Summarization |
+|---|------|------|------|------|------|------|------|------|------|---|
+| ace | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| acm | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| acq | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| aeb | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| afr | 0 | 4 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| ajp | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| aka | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| als | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| amh | 0 | 5 | 2 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| anp | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| apc | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| ara | 0 | 11 | 0 | 0 | 0 | 1 | 0 | 3 | 0 | 0 |
+| arb | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 2 | 0 | 0 |
+| arq | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| ars | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| ary | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| arz | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| asm | 0 | 3 | 1 | 0 | 0 | 1 | 0 | 2 | 1 | 0 |
+| ast | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| awa | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| ayr | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| azb | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| aze | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| azj | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| bak | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bam | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| ban | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bbc | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bel | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bem | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| ben | 0 | 9 | 1 | 0 | 0 | 1 | 0 | 2 | 1 | 0 |
+| bgc | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bhb | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bhd | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bho | 0 | 2 | 0 | 0 | 0 | 1 | 0 | 0 | 0 | 0 |
+| bjj | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bjn | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bns | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bod | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| bos | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bra | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| brx | 0 | 2 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bug | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| bul | 0 | 4 | 0 | 0 | 1 | 1 | 0 | 1 | 0 | 0 |
+| cat | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| ceb | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| ces | 0 | 2 | 0 | 0 | 1 | 0 | 0 | 1 | 0 | 0 |
+| chv | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| cjk | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| ckb | 0 | 1 | 0 | 0 | 0 | 0 | 0 | 1 | 0 | 0 |
+| cmn | 0 | 10 | 4 | 0 | 0 | 3 | 4 | 10 | 8 | 0 |
+| cmo | 0 | 2 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| code | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 19 | 0 | 0 |
```

Comment:
I would probably move all code to a separate table

Reply:
Let's do this in a separate PR?